### PR TITLE
Add Wasm Units tests, fix TSBindings null strings support

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type
- Bugfix
- Feature

## What is the current behavior?
Passing null strings as part of generated TSBinding transforms null pointers into `null` strings.

## What is the new behavior?
Null pointers to strings are now processed properly.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](../README.md#supported)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [x] Tests for the changes have been added (for bug fixes / features) (if applicable)
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: src/SamplesApp/SamplesApp.Shared/Samples/UnitTests/UnitTestsPage.xaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿<Page
+    x:Class="SamplesApp.Samples.UnitTests.UnitTestsPage"
```
Comment (agneszitte-nventive) [inline src/SamplesApp/SamplesApp.Shared/Samples/UnitTests/UnitTestsPage.xaml]: Indentation ;)

File: src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UnitTestsControl.xaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+﻿<UserControl
+    x:Class="Uno.UI.Samples.Tests.UnitTestsControl"
```
Comment (agneszitte-nventive) [inline src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UnitTestsControl.xaml]: Same format indentation

File: src/Uno.Foundation/Interop/TSInteropMarshaller.wasm.cs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+		public const UnmanagedType LPUTF8Str = (UnmanagedType)48;
+
+		/// <summary>
+		/// Prints the actual offsets of the structures present in <see cref="WindowManagerInterop"/> for debugging purposes.
+		/// </summary>
+		internal static void GenerateTSMarshallingLayouts()
+		{
+			Console.WriteLine("Generating layouts");
+
+			//foreach (var p in typeof(WindowManagerInterop).GetNestedTypes(System.Reflection.BindingFlags.NonPublic).Where(t => t.IsValueType))
```
Comment (agneszitte-nventive) [inline src/Uno.Foundation/Interop/TSInteropMarshaller.wasm.cs]: It's normal that this part is commented? Need it for later or forgot to remove it ?

File: src/SamplesApp/SamplesApp.Shared/Samples/UnitTests/UnitTestsPage.xaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:SamplesApp.Samples.UnitTests"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:ctl="using:Uno.UI.Samples.Tests"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Grid>
+		<ctl:UnitTestsControl />
```
Comment (MatFillion) [inline src/SamplesApp/SamplesApp.Shared/Samples/UnitTests/UnitTestsPage.xaml]: Suggestion : xmlns should be "controls" to follow https://nventive.visualstudio.com/Umbrella/_wiki/wikis/Umbrella.wiki?wikiVersion=GBwikiMaster&pagePath=%2FProgramming%20guidelines%2FGuidance%2FDevelopment%20standards%2FXAML&anchor=xaml-namespaces-(xmlns)

File: src/Uno.Foundation/Interop/TSInteropMarshaller.wasm.cs (hunk 3 above)
Comment (jeromelaban) [reply src/Uno.Foundation/Interop/TSInteropMarshaller.wasm.cs]: It's useful for debugging :)
